src/vm/win32threadpool.cpp

```diff
@@ -426,6 +428,11 @@ BOOL ThreadpoolMgr::Initialize()
     forceMax = GetForceMaxWorkerThreadsValue();
     MaxLimitTotalWorkerThreads = forceMax > 0 ? (LONG)forceMax : (LONG)GetDefaultMaxLimitWorkerThreads(MinLimitTotalWorkerThreads);
 
+    if (MinLimitTotalWorkerThreads == MaxLimitTotalWorkerThreads)
+    {
+        IsHillClimbingDisabled = true;
+    }
```

Comment:
What if the min and max are changed by the dev with calls to ThreadPool.SetMin/MaxThreads.  Could we go from a situation where hill climbing is disabled to one where it shouldn't be?

Reply:
True, this is unnecessary as well, I'll remove it